volume/mounts/lcow_parser.go

```diff
@@ -1,3 +1,5 @@
+// +build windows
```

Comment:
See above. It is because of the original idea to allow these on any platform. I could have renamed the files instead, but was trying to keep the change more minimal for now.